typos.toml

```diff
@@ -0,0 +1,28 @@
+[files]
+extend-exclude = [
+  "*.pbxproj",    # metadata file
+  "CHANGELOG.md", # To keep consistency between the commit history/PRs.
+]
+
+# Corrections take the form of a key/value pair. The key is the incorrect word
+# and the value is the correct word. If the key and value are the same, the
+# word is treated as always correct. If the value is an empty string, the word
+# is treated as always incorrect.
+
+# Match Whole Word - Case Sensitive
+[default.extend-identifiers]
+ser = "ser" # ron::ser - Serializer
+toi = "toi" # Time of intersention
```

Comment:
I feel like acronyms like these should be required to be UPPERCASE. Things like (lowercase) "toi" could easily be typos for "top" or "you".

Reply:
the problem is that "toi" is already used in bevy. 
I will move toi to extend-words to check for TOI and toi.

I will change the word in typos.toml for TOI.

Reply:
Interesting. Is there a way to opt-out the spellchecking on a line-by-line basis? Like with a comment or something (a la `// ts-expect-error`)?

Reply:
These might make sense for variables, but I feel like the bigger space for possible typos are in docs and such where it's generally more user-facing. In those scenarios, we probably want to mitigate false negatives.